crates/turborepo-lib/src/run/mod.rs

```diff
@@ -63,6 +65,14 @@ impl<'a> Run<'a> {
     }
 
     pub async fn run(&mut self) -> Result<i32> {
+        tracing::trace!(
+            platform = %TurboState::platform_name(),
+            start_time = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).expect("system time after epoch").as_micros(),
+            turbo_version = %TurboState::version(),
```

Comment:
Parameters supplied to the log using this syntax do not end up in the terminal, but they _do_ end up on the event in chrome tracing in the `args` object.